_posts/zh/2021-03-26-ALTF4-29.md

```diff
@@ -0,0 +1,69 @@
+
+{% include image.html src='https://media.alt-f4.blog/ALTF4/29/beacons.jpg' alt='围绕着组装机的插件效果分享塔，以及反之。' %}
+
+插件效果分享塔最好的两种使用方法是用建筑包围一个插件效果分享塔，或者用插件效果分享塔包围一个建筑。第一种情况有利于让那些昂贵的 3 级插件做更多的工作（多达每个插件 4 倍效果），第二种情况则是将一个建筑的速度远远超过其正常能力。在一台 3 级组装机内和周围塔有装满速度插件的情况下，建筑的速度是没无插件效果分享塔时的 2.6 倍，是完全没有插件的 8 倍。
+
+必须先说明一下插件效果分享塔不能做什么。插件效果分享塔不能包含产能插件，而且插件效果分享塔只能提升有插件插槽的建筑。所以，你不能把速度插件放在插件效果分享塔里，并期望你的钢炉工作得更快。然而，你的电炉却会加速。
+
+### 它们应该用在哪里？
+
+你可能会认为在插件效果分享塔中安装节能插件没有意义，因为插件效果分享塔本身就会消耗大量的电力，但是再考虑一下：因为一个插件效果分享塔总要消耗 480MW 的电力 如果你能在受影响的建筑上至少节省这么多电力，那么你就减少了整体的电力消耗。例如，三台组装机 3 的基本功耗是 1125MW。如果你在一个插件效果分享塔中放置两个节能插件 3，它们将减少附近建筑 50% 的电力消耗。由于 1,125 / 2 = 562.5MW，你可以节省 82.5MW。而且这只是在只有3台组装机的情况下，你可以很容易地在里面安装更多的设备。只是要注意机器有最低耗电 20% 的限制！
+
+插件效果分享塔中的速度插件更为常见。如果你有一些只是 *需要* 尽可能快地运行的东西（比如枯竭的抽油机），你可以用装满速度插件的插件效果分享塔包围它。如果你的工厂有一部分变得太大了，比如熔炉阵列，你可以在建筑物之间使用一排插件效果分享塔，以便缩小它们的体积。记住：你不需要巨型基地就可以来玩插件效果分享塔，不妨在你认为会有帮助的小角落里试试吧！
+
+{% include image.html src='https://media.alt-f4.blog/ALTF4/29/beaconed_furnaces.jpg' alt='插件效果分享塔覆盖的电炉与同容量的钢炉对比。' caption='请注意，左、右设计的冶炼吞吐量相同。' %}
+
+最后，我应该说节能插件会抵消其他插件的惩罚，所以你可以在你的炉子和采矿钻周围一起使用它们，以大大降低污染产出。最低耗电量（也等于污染系数）是建筑原值的 20%，而不是惩罚后的值，所以你的节能插件 3 在这里真正开始有用了。如果你把 14 个节能插件 3 和 10 个速度插件 3 混合在 12 个插件效果分享塔阵列中，你就可以完全抵消速度插件带来的污染增加，并且仍然可以获得 250% 的速度加成。
+
+看吧，我说过节能模块 3 很有用的！如果你有什么想说的，可以来 [Discord](https://discord.gg/AsXAwyV) 上找我，帮我整理一下未来的文章。在那之前，我们工厂车间见！
+
+## Mod 聚焦：[无传送带（No Belts）](https://mods.factorio.com/mod/no-belts) <author>oof2win2</author>
+
+*无传送带*是异星工厂的一个 Mod，它做了一些不是很多 Mod 敢做的事情：它删除了游戏的一个核心机制。正如你从标题中可能已经猜到了，这个 Mod 删除了你放置传送带的能力，而这通常是你开始自动化生产时最先放置下来的东西之一。取而代之的是，你可以在游戏中更早地放置物流机器人，基本上从你制造第一块绿色电路开始。
+
+一开始的磨练可能会觉得有点难，但它有一个回报：一切 ~~必须~~ 可以以机器人为基础。采矿，生产，一切都是。“总线”的概念可以扔掉，因为没有任何类似的东西是可能的（除非你用火车车厢和机械臂）。此外，你 **将会** 需要重新设计大量的布局，来使它们可以工作，因为连机器人都不能放置传送带。这意味着对于所有想使用的东西，你都可以享受重新设计的乐趣，而如果你设计得好的话，在未来还可能会派上用场！
+
+{% include image.html src='https://media.alt-f4.blog/ALTF4/29/beacon_base.jpg' alt='基于机器人的低密度设施' caption='传送带式低密度工厂转为机器人式。' href='https://media.alt-f4.blog/ALTF4/29/beacon_base_hr.jpg' %}
+
+上图是低密度生产设施。这是一个从使用传送带转换为使用机器人的蓝图的例子。在这种情况下，它是非常简单的，因为我只是使用了一个插件效果分享塔森林。然而，你可以创建任何你喜欢的设计 蓝图可以在[这里](https://media.alt-f4.blog/ALTF4/29/blueprint.txt)获得。它最多使用 6000 个物流机器人，以拥有与传送带版相同的产能，但当你不查看设施时，性能大多还可以。
+
+说到性能，你可能会有疑问。这是一个很有道理也可以理解的问题，机器人被一些人认为是 UPS 杀手。我也想过这个问题，所以我做了实验。我给 Alt-F4 团队发送了存档，他们也反馈了结果，所以下面是一个包含结果的表格。请注意，这里的“查看机器人”是指在一个地方查看 10 万个机器人，这是你在一般的游戏中不会遇到的。
+
+| 电脑配置                                    | FPS/UPS（查看机器人） | FPS/UPS（未查看机器人）|
+|--------------------------------------------|----------------------|-----------------------|
+| i7-8700, 16GB 内存 @ 3200MHz, GTX 1060      | 25                  | 60                    |
+| Ryzen 7 2700, 16GB 内存 @ 3200MHz, RTX 3080 | 35                  | 55                    |
+| Ryzen 5 3600, 32GB 内存 @ 3000MHz, GTX 1070 | 20                  | 57                    |
+| （笔记本）i7-5700, 8GB 内存, 集显            | 19                  | 27                    |
```

Comment:
@Conor12345 table not aligned have any effect?
ok for me if this is ok

Reply:
It is not a monospaced font so the table is not aligned in the .md file. Shoule be aligned in HTML page, as https://raw.githubusercontent.com/AlternativeFFFF/Alt-F4/master/_posts/zh/2021-02-12-ALTF4-23.md and https://alt-f4.blog/zh/ALTF4-23/

Reply:
Yep no issues with the table
![image](https://user-images.githubusercontent.com/34628734/112629848-b4ebf580-8e2c-11eb-83d5-4b5530454d9c.png)